cloud-pubsub/hpa/hpa.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: autoscaling/v2beta1
```

Comment:
can you rename hpa/ dir to manifests/

Reply:
Should I also move deployment yaml to the new folder? That would break the existing tutorial between merging PR and updating the link in there.

Reply:
in that case, let's move this to ../ and call it horizontalpodautoscaler.yaml

Reply:
or pubsub-hpa.yaml.

Reply:
done